src/charactersheet/viewmodels/dm/initiative-tracker/index.html

```diff
@@ -0,0 +1,320 @@
+  <div style="hieight: 100px" class="loader-wrapper"><div style="width: 75px; height:75px; margin: 25px auto" class="loader"></div></div>
+  <!-- /ko -->
+  <!-- ko if: loaded(), completeOn: 'render' -->
+  <!-- ko ifnot: isConnectedToParty -->
+  <div class="row">
+    <div class="col-xs-12">
+      <div class="detail-place-holder text-center">
+        <p class="lead text-muted heading">You are not connected to a Party.</p>
+        <p class="text-muted">
+          Create or join a Party with friends to see their characters here!<br/>
+          Learn more about Parties <a href="https://adventurerscodex.com/collaboration.html#partyDashboard" target="_blank">here</a>.
+        </p>
+      </div>
+    </div>
+  </div>
+  <!--/ko-->
+  <!-- ko if: isConnectedToParty -->
+  <div class="row">
+    <div class="col-xs-12 col-md-7 col-lg-8">
+      <table class="table table-responsive table-ac-bordered table-hover">
+        <thead class="ac-table-header">
+          <tr>
+            <th class="col-xs-7 col-md-8 col-lg-9">
+              <!-- ko if: state() === State.called -->
+              Participants (unsorted)
+              <!-- /ko -->
+              <!-- ko if: state() === State.started -->
+              Initiative Order
+              <!-- /ko -->
+            </th>
+            <th class="col-xs-4 col-md-3 col-lg-2 text-center">
+              <button
+                class="btn btn-xs btn-info"
+                data-bind="click: rollForInitiative, enable: canReRoll"
+                type="button"
+              >
+                Roll for Initiative
+              </button>
+            </th>
+            <th class="col-xs-1 text-right">
+              <a href="#" data-bind="click: collapseAll">
+                <span class="fa fa-minus"></span>
+              </a>
+            </th>
+          </tr>
+        </thead>
+        <!-- ko ifnot: loaded(), completeOn: 'render' -->
+        <tbody>
+          <tr>
+            <td colspan="3">
+              <div class="loader-wrapper">
+                <div class="loader"></div>
+              </div>
+            </td>
+          </tr>
+        </tbody>
+        <!-- /ko -->
+        <!-- ko if: loaded(), completeOn: 'render' -->
+        <tbody>
+          <!-- ko template: { name: 'participant-row', foreach: participants } -->
+          <!-- /ko -->
+        </tbody>
+        <tfoot>
+          <tr class="collapseRow">
+            <td colspan="3">
+              <div id="add-initiative" class="collapse add-card">
+                <initiative-form params="{
+                  containerId:'add-initiative',
+                  show: $component.displayAddForm,
+                  flip: $component.toggleShowAddForm,
+                  remainingPlayers: $component.remainingPlayers,
+                 }"></initiative-form>
+              </div>
+              <!-- ko ifnot: $component.displayAddForm -->
+              <center>
+                <button
```

Comment:
This button can be clicked anywhere on the table row, so if you click on the seemingly empty space waaay to the right, this form comes up. 
Maybe the button shouldn't span the whole table?